examples/App.config

```diff
@@ -1,6 +1,14 @@
-﻿<?xml version="1.0" encoding="utf-8" ?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
     <startup> 
         <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.8" />
     </startup>
+  <runtime>
```

Comment:
Does the example app still work if this change is removed? I'm curious if this will be a required change for any consumers of this library. (Or if it still works fine, if we can remove this from the commit)

Reply:
Thank you! I've updated the PR